### PR TITLE
databases-week-1-muhammet-emin-alper

### PR DESCRIPTION
**Pull request by eminalpernl**

### CODE REVIEW

**Comment by ArcoMul** [comment]
Hi @eminalpernl, thank you for submitting the homework. Sadly I'm not able to run your code. A `package.json`/`package-lock.json` is missing, and you are using `import` in app.js, but as far as I know this is not supported without `webpack` or `vite` or something like that. Could you share how you start the application? I tried it with node 18.14.0 and 19.6.0

File: Week1/Ex-1/app.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+querier(`
+CREATE TABLE IF NOT EXISTS Room (
+  room_no INT AUTO_INCREMENT PRIMARY KEY,
+  room_name VARCHAR(255) NOT NULL,
+  floor_number INT NOT NULL
+)`, 'Room table created successfully');
+
+// Create the Meeting table
+querier(`
+CREATE TABLE IF NOT EXISTS Meeting (
```
Comment (ArcoMul) [inline Week1/Ex-1/app.js:32]: Does exercise 1 execute correctly for you? I'm getting the following error:

```
code: 'ER_INVALID_DEFAULT',
  errno: 1067,
  sqlMessage: "Invalid default value for 'ending_time'",
  sqlState: '42000',
  index: 0,
  sql: '\n' +
    'CREATE TABLE IF NOT EXISTS Meeting (\n' +
    '  meeting_no INT AUTO_INCREMENT PRIMARY KEY,\n' +
    '  meeting_title VARCHAR(255) NOT NULL,\n' +
    '  starting_time TIMESTAMP NOT NULL,\n' +
    '  ending_time TIMESTAMP NOT NULL,\n' +
    '  room_no INT NOT NULL,\n' +
    '  FOREIGN KEY (room_no) REFERENCES Room(room_no)\n' +
    ')'
 ```

File: Week1/Ex-2/app.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  console.log('Connected to World DB as id ' + con.threadId);
+});
+
+questions.forEach((el) => {
+  con.query(el.sql, function (error, results) {
+    if (error) throw error;
+    console.log(el.question);
+    console.table(results);
+  });
+});
```
Comment (ArcoMul) [inline Week1/Ex-2/app.js:25]: Very nice approach with the questions array, and querying and loggin the results in this manner!

**Review by ArcoMul — CHANGES_REQUESTED** [review]
Hi @eminalpernl, thank you for proving the missing files! It's running fine now, only with exercise 1 I'm getting an SQL error, exercise 2 runs perfectly and looks good to me! :-)

File: Week1/Ex-1/app.js (hunk 1 above)
Comment (eminalpernl) [reply Week1/Ex-1/app.js:32]: I'm getting this when I run the Ex-1:

the database with the name of meetup deleted
the database with the name of meetup created successfully
Connected to "meetup"
Invitee table created successfully
Room table created successfully
Meeting table created successfully
Invitee table updated
Room table updated
Meeting table updated

**Pull request closed without merging**